Simplify Linux install overview

richlander opened the pull request:
I had a few goals here.

- Put the focus on getting distro-specific install instructions.
- Elevate the duality of official distro archives and packages.microsoft.com and prioritize them in that order.
- Remove all the support information. It is super confusing. Replace it with a link to our OS support pages on GH.

[internal preview link](https://review.docs.microsoft.com/en-us/dotnet/core/install/linux?branch=pr-en-us-29462)

richlander committed:
Simplify Linux install overview

richlander committed:
Add support statement

richlander committed:
Add tarball option

adegeo commented on docs/core/install/linux.md:
```diff
@@ -1,200 +1,68 @@
-- A ✔️ indicates that the version of Ubuntu or .NET is still supported.
-- A ❌ indicates that the version of Ubuntu or .NET isn't supported on that Ubuntu release.
-- When both a version of Ubuntu and a version of .NET have ✔️, that OS and .NET combination is supported.
+## Other Distros
```
I don't think we use the word distro in docs
```suggestion
## Other distributions
```

adegeo commented on docs/core/install/linux.md:
```diff
@@ -1,200 +1,68 @@
-| ✔️ [20.04 (LTS)](linux-ubuntu.md#2004) | ✔️ 3.1        | ✔️ 5.0 | ✔️ 6.0 |
-| ✔️ [18.04 (LTS)](linux-ubuntu.md#1804) | ✔️ 3.1        | ✔️ 5.0 | ✔️ 6.0 |
-| ✔️ [16.04 (LTS)](linux-ubuntu.md#1604) | ✔️ 3.1        | ✔️ 5.0 | ✔️ 6.0 |
+Installation information is als provided for other distributions.
```
bug
```suggestion
Installation information is also provided for other distributions.
```

adegeo reviewed (changes requested):
I'm willing to try this since it simplifies the content. However, note that we do have a lot of views on this content (almost 50k a month) and it has good ratings. This would indicate that it is satisfying the needs of the readers.

There are a few things to fix up before merging.

richlander commented:
I don't doubt that some people like the current format. It has a lot of information and it is (mostly) accurate. I have to believe that others are overwhelmed by all the information. There is a lot more information than is needed for folks to be successful. Last,  all of this support information take a lot of care and feeding. Ideally, we can just rely on it in release notes (which uses a simpler format) and call it good.

Yes, let's see if we can maintain or improve the rating!

richlander committed:
Apply suggestions from code review

Co-authored-by: Andy (Steve) De George <67293991+adegeo@users.noreply.github.com>

richlander committed:
Fix linter error

From linter

Error: docs/framework/wcf/feature-details/interoperability-and-integration.md:35:67 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 3]

richlander committed:
Update linux.md

richlander committed:
Merge branch 'main' into richlander-linux-overview

richlander committed:
Update linux.md

richlander commented:
I believe this topic is ready to go @adegeo.

richlander committed:
Update linux.md

richlander committed:
Update linux.md

richlander commented:
I believe this topic is ready to go @adegeo.

richlander committed:
Update linux.md

adegeo committed:
Update linux.md

richlander commented:
Thanks @adegeo!
